lib/wibox/widget/slider.lua

```diff
@@ -381,7 +381,7 @@ function slider:draw(_, cr, width, height)
 
     if bar_active_color and type(bar_color) == "string" and type(bar_active_color) == "string" then
         local bar_active_width = active_rate * (width - x_offset - right_margin)
-            - (handle_width - handle_border_width/2) * (active_rate - 0.5)
+            - math.floor(handle_width - handle_border_width/2) * (active_rate - 0.5)
```

Comment:
why here and next you making floor() not of whole equation but only to some part?

Reply:
done